pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml

```diff
@@ -11,7 +11,53 @@
       url_secondary: '/ai#ai-curricula',
       aria_label_primary: hoc_s(:aria_label_call_to_action_ai_101),
       aria_label_secondary: hoc_s(:aria_label_call_to_action_curriculum_ai_101),
-    },
+      new: true
+    }
+  ]
+  if !!DCDO.get('curriculum-launch-2024', false)
+    block_items +=[
+      {
+        overline: hoc_s(:module_grade_3_5_teachers),
+        title: hoc_s(:module_label_teaching_how_ai_makes_decisions),
+        image: '/images/self-paced-pl-tile-how-ai-makes-decisions.jpg',
+        curriculum: hoc_s(:curriculum_name_how_ai_makes_decisions),
+        duration: hoc_s(:module_duration_1_hr),  
+        prerequisites: hoc_s(:module_prerequisites_none),
+        url_primary: CDO.studio_url("/courses/elementaryai-2024"),
+        url_secondary: CDO.studio_url('/s/k5-ai-data-2024'),
+        aria_label_primary: hoc_s(:aria_label_call_to_action_teaching_how_ai_makes_decisions),
+        aria_label_secondary: hoc_s(:aria_label_call_to_action_curriculum_how_ai_makes_decisions),
+        new: true
+      },
+      {
+        overline: hoc_s(:module_grade_3_5_teachers),
+        title: hoc_s(:module_label_teaching_elementary_game_design),
+        image: '/images/self-paced-pl-tile-elementary-game-design.jpg',
+        curriculum: hoc_s(:curriculum_name_elementary_game_design),
+        duration: hoc_s(:module_duration_1_hr_30_minutes),
+        prerequisites: hoc_s(:module_prerequisites_none),
+        url_primary: CDO.studio_url("/courses/3-5gamedesign-2024"),
+        url_secondary: CDO.studio_url('/courses/3-5gamedesign-2024'),
```

Comment:
This still doesn't feel right to me. It doesn't affect translations, so I'll leave it as-is for now and we can update. Jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1999